### PR TITLE
Improve issue templates

### PR DESCRIPTION
**Pull request by erbridge**
This sets the "needs triage" label on all new issues to support that as a new workflow.

It also created a new technical debt template to reduce the amount of skipping outside of the templates being done.

### CODE REVIEW

**Comment by kimadactyl** [comment]
Is `tech debt` and `chore` the same thing? Do we need both?

**Comment by erbridge** [comment]
No, they're not the same thing imo, but I would happily burn "chore" as a label because nobody uses it to mean the same thing.

**Comment by kimadactyl** [comment]
Yeah, `chore` is used by release drafter to mean "goes in the maintance section" - will approve this and leave it to you if you want to update the release drafter config to use `tech debt` instead as part of this PR

**Review by kimadactyl — APPROVED** [review]

**Comment by kimadactyl** [comment]
Oh right it's on automerge, lol. Np either way :)

**Pull request merged**
